PRChecker/Shared/ContentView.swift

```diff
@@ -8,9 +8,43 @@
 import SwiftUI
 
 struct ContentView: View {
+    @State var showLogin = true
     var body: some View {
-        Text("Hello, world!")
-            .padding()
+        GeometryReader { geometry in
+            HStack {
+                VStack {
+                    Spacer()
+                    Button {
+                        showLogin.toggle()
+                    } label: {
+                        Text(.init(systemName: "person"))
+                            .font(.system(size: 45))
+                            .foregroundColor(.secondary)
+                    }
+                    .buttonStyle(.link)
+                    .popover(isPresented: $showLogin, arrowEdge: .trailing) {
+                        LoginView {
+                            // TODO: Refresh
+                        }
+                    }
+
+                    Button {
+                        // TODO: Settings Page
+                    } label: {
+                        Text(.init(systemName: "gearshape"))
+                            .font(.system(size: 45))
+                            .foregroundColor(.secondary)
+                    }
+                        .buttonStyle(.link)
+                }
+                .frame(alignment: .leading)
```

Comment:
Ah, I haven't understood the difference between `VStack().frame(alignment: .leading)` and `VStack(alignment: .leading)`..

Reply:
Oh... neither do I lol... Let me try it out

Reply:
Ah, `VStack(alignment: .leading)` sets the internal alignment to `.leading` and `.frame(alignment: .leading)` sets the alignment of the VStack itself.

...either way when I tested it out deleting both didn't change the UI so I'll simplify it by removing it.

Reply:
> ...either way when I tested it out deleting both didn't change the UI so I'll simplify it by removing it.

Unexpected ending😂